### PR TITLE
Add manifests needed for prescriptions-refresh

### PR DESCRIPTION
**Pull request by fridex**
## Description

Introduce [prescriptions-refresh](https://github.com/thoth-station/prescriptions-refresh-job) CronWorkflow.

### CODE REVIEW

**Comment by fridex** [comment]
Ready to be reviewed. PTAL.

To make this work properly in stage and prod, Kebechet secrets will need to be deployed into the mentioned namespaces as well.

/assign @harshad16

File: prescriptions-refresh-job/base/overlays/test/job-generate-name.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+- op: add
+  path: /metadata/namespace
+  value: "thoth-middletier-stage"
```
Comment (harshad16) [inline prescriptions-refresh-job/base/overlays/test/job-generate-name.yaml:3]:
```suggestion
  value: "thoth-test-core"
```

**Review by harshad16 — CHANGES_REQUESTED** [review]
- Remove **overlays** from the **base** directory.
- change **cnv-prod** to **zero-prod**

**Comment by fridex** [comment]
@harshad16 Adjusted based on the review, thanks 👍🏻 PTAL

File: prescriptions-refresh-job/overlays/ocp4-stage/job-generate-name.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+- op: add
+  path: /metadata/namespace
+  value: "thoth-middletier-stage"
```
Comment (harshad16) [inline prescriptions-refresh-job/overlays/ocp4-stage/job-generate-name.yaml:3]:
```suggestion
  value: "thoth-infra-stage"
```

File: prescriptions-refresh-job/overlays/test/job-generate-name.yaml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+- op: add
+  path: /metadata/namespace
+  value: "thoth-middletier-stage"
```
Comment (harshad16) [inline prescriptions-refresh-job/overlays/test/job-generate-name.yaml:3]:
```suggestion
  value: "thoth-infra-stage"
```

File: prescriptions-refresh-job/overlays/test/job-generate-name.yaml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+- op: add
+  path: /metadata/namespace
+  value: "thoth-infra-stage"
```
Comment (harshad16) [inline prescriptions-refresh-job/overlays/test/job-generate-name.yaml:3]:
```suggestion
  value: "thoth-test-core"
```

File: prescriptions-refresh-job/overlays/zero-prod/job-generate-name.yaml [hunk 5]
```diff
@@ -0,0 +1,3 @@
+- op: add
+  path: /metadata/namespace
+  value: "thoth-middletier-prod"
```
Comment (harshad16) [inline prescriptions-refresh-job/overlays/zero-prod/job-generate-name.yaml:3]:
```suggestion
  value: "thoth-infra-prod"
```

**Review by harshad16 — COMMENTED** [review]
lgtm
thanks

**Pull request merged**